extension/scripts/features/hidegymhighlight/ttHideGymHighlight.css

```diff
@@ -0,0 +1,7 @@
+body:not(.tt-mobile) .tt-available {
```

Comment:
light theme should also have `:not(.dark-mode)`

Reply:
Done.

Reply:
Added.